apps/etterlatte-saksbehandling-ui/client/src/components/behandling/avkorting/Avkorting.tsx

```diff
@@ -37,16 +37,14 @@ export const Avkorting = ({
   const visSanksjon = useFeaturetoggle(FeatureToggle.sanksjon)
   const inntektNesteAarBryter = useFeaturetoggle(FeatureToggle.validere_aarsintnekt_neste_aar)
 
+  const virkAar = () => new Date(virkningstidspunkt(behandling).dato).getFullYear()
```

Comment:
Kan ikke denne skrives slik?
```suggestion
  const virkAar = new Date(virkningstidspunkt(behandling).dato).getFullYear()
```